news/_posts/translation/2020-05-14-do-you-speak-prestashop-april-2020.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – April 2020 edition"
+subtitle: "Where you'll learn that PrestaShop is fully available in Danish"
+date:  2020-05-14 09:10:11
+authors: [JuneByun]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+ 
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in April.
+ 
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+ 
+## Project news
+ 
+ 
+### First news, new content manager :tada:
+ 
+I would like to take this opportunity to present myself to all of you! My name is June, I joined the PrestaCrew as a Content manager last week, and I will be in charge of the Crowdin PrestaShop translation projects in the next several months. I am very excited about discovering this tool and am delighted to see that many of you from across the globe dedicate their time to these projects. I want to say thank you for your hard work, your contributions do not only help us but to anyone who is using or will be using PrestaShop ;) I look forward to communicating with you guys, if you have any questions do not hesitate to contact me anytime, I would be more than happy to answer you :smiley:
```

Comment:
In general, please try to be very neutral and incluse. The PrestaShop company is a contributor to the PrestaShop project (yes, it's confusing). That's why I suggest to remove "us" in this paragraph.

```suggestion
I would like to take this opportunity to present myself to all of you! My name is June, I joined the PrestaCrew as a Content manager last week, and I will be in charge of the Crowdin PrestaShop translation projects in the next several months. I am very excited about discovering this tool and am delighted to see that many of you from across the globe dedicate their time to these projects. I want to say thank you for your hard work, your contributions help anyone who is using or will be using PrestaShop ;) I look forward to communicating with you guys, if you have any questions do not hesitate to contact me anytime, I would be more than happy to answer you :smiley:
```

Reply:
Ok, I'll keep that in mind next time!